client/src/AppRoutes.tsx

```diff
@@ -1,17 +1,47 @@
-import React, { lazy, Suspense } from 'react'
-import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import React, { lazy, Suspense, useEffect, useState } from 'react'
+import { BrowserRouter, Route, Routes, Navigate } from 'react-router-dom'
 import Notion from './Notion'
-import { Loading } from './components/Misc'
-// import Manage from './Manage'
-const Manage = lazy(async () => await import('./Manage'))
+import Substack from './Substack'
+import { Loading, LoadingScreen } from './components/Misc'
+import { buildClient, EWSTypes } from './api'
+import { getSld, getSubdomain } from './utils'
+const Manage = lazy(async () => await import('./manage'))
+
+const client = buildClient()
+const sld = getSld()
+const subdomain = getSubdomain()
 
 const AppRoutes: React.FC = () => {
+  const [ewsType, setEwsType] = useState<number>()
+
+  useEffect(() => {
+    client
+      .getEwsType(sld, subdomain)
+      .then(e => { setEwsType(e) })
+    .catch(console.error)
+  }, [])
+
+  if (ewsType === undefined) {
+    return <LoadingScreen />
+  }
+
   return (
     <BrowserRouter>
       <Routes>
         <Route path={'/manage'} element={
-          <Suspense fallback={<Loading/>}><Manage /></Suspense>} />
-        <Route path='/*' element={ <Notion />} />
+          <Suspense fallback={<Loading/>}>
+            <Manage ewsType={ewsType} />
+          </Suspense>} />
+        <Route
+          path='/*'
+          element={
+            ewsType === EWSTypes.EWS_UNKNOWN
```

Comment:
might need to treat unknown as notion. I will experiment and fix later

Reply:
Done